### PR TITLE
Create starter Django project and define apps

### PR DESCRIPTION
**Pull request by gogolmariana**
Created new **forum** Django project and added **users**, **projects**, **profiles**, **communications**, **dashboard** apps to the project.

### CODE REVIEW

**Review by mehalyna — CHANGES_REQUESTED** [review]
The `.idea` folder was included in the pull request. This folder contains project-specific configuration files generated by PyCharm and should not be committed to the repository. Here are the steps to exclude the `.idea` folder from the PR and future commits:

1. **Remove the `.idea` Folder from the PR:**
   - The `.idea` folder should not be included in the repository as it contains personal project settings that can differ between developers.

2. **Add `.idea` to `.gitignore`:**
   - To prevent this folder from being included in future commits, you can add it to the `.gitignore` file (if it isn't already).
   
   Example of adding `.idea` to `.gitignore`:
   ```
   .idea/
   ```

3. **Remove `.idea` from Git Tracking:**
   - Even after adding `.idea` to the `.gitignore` file, Git may still track the folder if it has already been committed. To untrack it, follow these steps:

   Run the following commands in the terminal:
   ```bash
   git rm -r --cached .idea
   git commit -m "Remove .idea from tracking"
   ```


**Instructions to Exclude `.idea` in PyCharm:**

1. **Edit `.gitignore` Directly in PyCharm:**
   - Open the `.gitignore` file in PyCharm (in the root directory).
image

   - Add the following line to the file:
     ```
     .idea/
     ```
   - Save the file.

2. **Remove the Folder from Git Tracking in PyCharm:**
   - Right-click on the `.idea` folder in the PyCharm project view.
   - Select **Git** > **Repository** > **Untrack** (or use the Git terminal inside PyCharm and run `git rm -r --cached .idea`).
   - Commit the changes by removing the `.idea` folder from Git tracking.

File: forum/forum/settings.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-b*jlp6t^_fvl_uu++x#!9rbi9m0!)7hzokn*y@82uqj)^+py+8'
+
```
Comment (VovaD199) [inline forum/forum/settings.py:24]: You have to hide the SECRET_KEY in .env files

File: forum/forum/settings.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+# Database
+# https://docs.djangoproject.com/en/5.1/ref/settings/#databases
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": "postgres",
+        "USER": "postgres",
```
Comment (VovaD199) [inline forum/forum/settings.py:86]: You should hide your database connection details to .env files

File: forum/forum/urls.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
```
Comment (VovaD199) [inline forum/forum/urls.py:21]: Please add the paths to your applications.

**Comment by gogolmariana** [comment]
> The `.idea` folder was included in the pull request. This folder contains project-specific configuration files generated by PyCharm and should not be committed to the repository. Here are the steps to exclude the `.idea` folder from the PR and future commits:
> 
>     1. **Remove the `.idea` Folder from the PR:**
>        
>        * The `.idea` folder should not be included in the repository as it contains personal project settings that can differ between developers.
> 
>     2. **Add `.idea` to `.gitignore`:**
>        
>        * To prevent this folder from being included in future commits, you can add it to the `.gitignore` file (if it isn't already).
>        
>        Example of adding `.idea` to `.gitignore`:
>        ```
>        .idea/
>        ```
> 
>     3. **Remove `.idea` from Git Tracking:**
>        
>        * Even after adding `.idea` to the `.gitignore` file, Git may still track the folder if it has already been committed. To untrack it, follow these steps:
>        
>        Run the following commands in the terminal:
>        ```shell
>        git rm -r --cached .idea
>        git commit -m "Remove .idea from tracking"
>        ```
> 
> 
> **Instructions to Exclude `.idea` in PyCharm:**
> 
>     1. **Edit `.gitignore` Directly in PyCharm:**
>        
>        * Open the `.gitignore` file in PyCharm (in the root directory).
>          image
>        * Add the following line to the file:
>          ```
>          .idea/
>          ```
>        * Save the file.
> 
>     2. **Remove the Folder from Git Tracking in PyCharm:**
>        
>        * Right-click on the `.idea` folder in the PyCharm project view.
>        * Select **Git** > **Repository** > **Untrack** (or use the Git terminal inside PyCharm and run `git rm -r --cached .idea`).
>        * Commit the changes by removing the `.idea` folder from Git tracking.

Ok.

I am going to remove from **.gitignore**

`
option (not recommended) you can uncomment the following to ignore the entire idea folder.
.idea
`

File: forum/forum/settings.py (hunk 2 above)
Comment (gogolmariana) [reply forum/forum/settings.py]: Done.

File: forum/forum/urls.py (hunk 3 above)
Comment (gogolmariana) [reply forum/forum/urls.py:21]: Added paths to applications.

**Review by mehalyna — APPROVED** [review]

**Review by Chelakhovl — APPROVED** [review]

**Review by AnnKvytko — APPROVED** [review]

**Pull request merged**